Cargo.toml

```diff
@@ -3,6 +3,7 @@ resolver = "2"
 members = [
   "crates/cairo-lint-cli",
   "crates/cairo-lint-core",
+  "crates/cairo-lint-dev",
```

Comment:
this is basicaly the [`xtask`](https://github.com/matklad/cargo-xtask) pattern, so maybe let's use the popular name for it?